src/test/java/tests/CollaboratorServiceExTests.java

```diff
@@ -0,0 +1,62 @@
+package tests;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import github.CollaboratorServiceEx;
+import org.eclipse.egit.github.core.RepositoryId;
+import org.eclipse.egit.github.core.User;
+import org.eclipse.egit.github.core.client.GitHubClient;
+import org.junit.Test;
+import org.mockserver.client.server.MockServerClient;
+import org.mockserver.integration.ClientAndServer;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockserver.model.HttpRequest.request;
+import static org.mockserver.model.HttpResponse.response;
+
+public class CollaboratorServiceExTests {
+
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
You can use ExpectedException for this instead, see [here](https://github.com/HubTurbo/HubTurbo/blob/master/src/test/java/tests/FilterParserTests.java#L19-L26).